integrations/visual_studio/cmake.rst

```diff
@@ -0,0 +1,196 @@
+    1> Working directory: C:\Users\user\CMakeBuilds\df6639d2-3ef2-bc32-abb3-2cd1bdb3c1ab\build\x64-Debug
+    1> -- The CXX compiler identification is MSVC 19.12.25831.0
+    1> -- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.12.25827/bin/HostX64/x64/cl.exe
+    1> -- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.12.25827/bin/HostX64/x64/cl.exe -- works
+    1> -- Detecting CXX compiler ABI info
+    1> -- Detecting CXX compiler ABI info - done
+    1> -- Detecting CXX compile features
+    1> -- Detecting CXX compile features - done
+    1> CMake Error at CMakeLists.txt:4 (include):
+    1>   include could not find load file:
+    1>
+    1>     C:/Users/user/CMakeBuilds/df6639d2-3ef2-bc32-abb3-2cd1bdb3c1ab/build/x64-Debug/conanbuildinfo.cmake
+
+As expected, our *CMakeLists.txt* is using an ``include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)``, and that file doesn't exist yet,
+because Conan has not yet installed the dependencies of this project. Visual Studio 2017 uses different build folders for each
+configuration. In this case, the default configuration at startup is ``x64-Debug``. This means that we need to install the
+dependencies that match this configuration. Assuming that our default profile is using Visual Studio 2017 for x64 (it should typically be
+the default one created by Conan if VS2017 is present), then all we need to specify is the ``-s build_type=Debug`` setting:
+
+.. code-block:: bash
+
+    $ conan install . -s build_type=Debug -if=C:\Users\user\CMakeBuilds\df6639d2-3ef2-bc32-abb3-2cd1bdb3c1ab\build\x64-Debug
+
+Now, you should be able to regenerate the CMake project from the IDE, Menu->CMake, build it, select the "example" executable to run, and run
+it.
+
+Now, let's say that you want to build the Release application. You switch configuration from the IDE, and then the above error happens
+again. The dependencies for Release mode need to be installed too:
+
+.. code-block:: bash
+
+    $ conan install . -if=C:\Users\user\CMakeBuilds\df6639d2-3ef2-bc32-abb3-2cd1bdb3c1ab\build\x64-Release
+
+The process can be extended to x86 (passing ``-s arch=x86`` in the command line), or to other configurations. For production usage,
+Conan **profiles** are highly recommended.
+
+Using cmake-conan
```

Comment:
This shouldn't be here at all. At most in the ``cmake`` integrations section

Reply:
Also beware the ``cmake-conan`` might not be using or ready for 2.0 at all. Better drop it at the moment

Reply:
Dropped